.gitignore

```diff
@@ -1,9 +1,7 @@
 /coverage
-/demo/dist
-/es
-/lib
+/build-demo
+/build-lib
 /node_modules
-/umd
```

Comment:
Would it be hard to keep the UMD around for a little while? Not essential, but it might help the staging of things.

Reply:
I think it is easier if things live under the directories listed in `config/paths.js` https://github.com/hubmapconsortium/vitessce/pull/495/files#diff-e389ed4107fbd6139ae8b315cb04e04dR66 since these directories are used in a few places (for example to compute file sizes https://github.com/hubmapconsortium/vitessce/pull/495/files#diff-6d31360883086ae2d6d1dc8e0857074cR74 )

Reply:
Also, packages that import vitessce code should be depending on the "main", "module", etc path definitions in the package.json right? If so, these package.json updates should suffice https://github.com/hubmapconsortium/vitessce/pull/495/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15

Reply:
And just to clarify, a UMD build is still being generated, it would just live within the `build-lib/umd/` subdirectory